spec.html

```diff
@@ -357,10 +357,25 @@ <h1>Runtime Semantics: HostInitializeSyntheticRealm ( _realm_ )</h1>
 				operation used to inform the host of any newly created realms from
 				the Realm constructor. Its return value is not used, though it may
 				throw an exception. The idea of this hook is to initialize host
-				data structures related to the Realm, e.g., for module loading. It
-				is not expected that this hook would add properties to the Realm's
-				global object.
+				data structures related to the Realm, e.g., for module loading.
 			</p>
+			<p>
+				The host may use this hook to add properties to the Realm's global
+				object. Those properties must each be configurable to provide
+				platform capabilities with no authority to cause side effects such
+				as I/O or mutation of values that are shared across different
+				realms within the same host environment.
+			</p>
+			<emu-note>
+				<p>
+					This specification does not recommend any specific addition.
+					In the Web embedding, HTML and WebIDL will specify which
+					interfaces are included. The Web Platform and Web-like
+					environments may decide to include `setTimeout`, `EventTarget`,
```

Comment:
AFAIK setTimeout is IO. And as I explained in #284 I strongly against the host being able to add things into the realm without being able to have a clean version. This also hurts the API to be cross-platform safe.

Reply:
This is just an editorial note and we can remove the example. 

Anyway, please keep the discussion to the respective issue.